pkg/services/ldap/ldap.go

```diff
@@ -95,7 +96,7 @@ func (server *Server) Dial() error {
 	if server.Config.RootCACert != "" {
 		certPool = x509.NewCertPool()
 		for _, caCertFile := range strings.Split(server.Config.RootCACert, " ") {
-			pem, err := ioutil.ReadFile(caCertFile)
+			pem, err := ioutil.ReadFile(filepath.Clean(caCertFile))
```

Comment:
After reviewing the gosec [issue thread](https://github.com/securego/gosec/issues/439), I think we should silence this warning, since `filepath.Clean` doesn't resolve any security issue and any security concerns around the folder path should be resolved when originally constructing it.